services/121-service/src/payments/fsp-integration/excel/excel.service.spec.ts

```diff
@@ -81,7 +81,7 @@ describe('ExcelService', () => {
     excelService = moduleRef.get<ExcelService>(ExcelService);
   });
 
-  it.skip('should find and return the matching reconciliation record for a given registration', async () => {
+  it('should find and return the matching reconciliation record for a given registration', async () => {
```

Comment:
I think these tests no longer work like this, because they test joinRegistrationsAndImportRecords which is by now only public for the sake of the unit test, so it actually shouldn't be public any more.